### DISCUSSION TITLE
Enhancement: Quick Auto Setup

### DISCUSSION THREAD

**Discussion by snowwing** [linuxserver/Heimdall#728] [Ideas]
I would be nice if there was some kind of auto discovery to setup initial applications (foundation and enhanced).

It could scan the common/default ports on the same IP it is running on and if it finds a known application running on that port, it sets it up. And maybe it could detect if it is running in a docker container for enhanced application setup.

**Comment by Cerbis** [comment]
+1 - i like this idea. i'm currently using the Traefik-API to add new services to my own lil dashboard. Would be nice if Heimdall could support that.

**Comment by nicoulaj** [comment]
@Cerbis: I also use traefik, would you mind sharing your script ?
